ts/store/reducers/wallet/outcomeCode.ts

```diff
@@ -145,6 +145,9 @@ export const lastPaymentOutcomeCodeSelector = (
   state: GlobalState
 ): OutcomeCodeState => state.wallet.lastPaymentOutcomeCode;
 
+export const paymentSelector = (state: GlobalState): any =>
```

Comment:
Since this selector returns another section of the state I guess that would be more appropriate moving it to the appropriate reducer definition -> [ts/store/reducers/wallet/payment.ts](https://github.com/pagopa/io-app/blob/master/ts/store/reducers/wallet/payment.ts)

what do you think?

Reply:
Yep!
Fixed as suggested ✅